<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_window</class>
 <widget class="QMainWindow" name="login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>579</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>383</width>
    <height>579</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>383</width>
    <height>579</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="login_header">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <widget class="QPushButton" name="login_close_button">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/images/cross.png</normaloff>:/images/images/cross.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="login_minimize_button">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/images/minimize.png</normaloff>:/images/images/minimize.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="login_body">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(21, 21, 21);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="login_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>321</width>
            <height>351</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(21, 21, 21);</string>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="usesScrollButtons">
           <bool>true</bool>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Log in</string>
           </attribute>
           <widget class="QLineEdit" name="login_username">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>70</y>
              <width>171</width>
              <height>31</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="login_password">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>150</y>
              <width>171</width>
              <height>32</height>
             </rect>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
           <widget class="QPushButton" name="login_button">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>230</y>
              <width>151</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Noto Serif</family>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 173, 103);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Log In</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Sign in</string>
           </attribute>
           <widget class="QLineEdit" name="signin_username">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>50</y>
              <width>171</width>
              <height>32</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="signin_password">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>120</y>
              <width>171</width>
              <height>32</height>
             </rect>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="signin_confirm_password">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>160</y>
              <width>171</width>
              <height>32</height>
             </rect>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Confirm Password</string>
            </property>
           </widget>
           <widget class="QPushButton" name="signin_button">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>230</y>
              <width>151</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Noto Serif</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 173, 103);</string>
            </property>
            <property name="text">
             <string>Sign In</string>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QFrame" name="login_icon">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>0</y>
            <width>91</width>
            <height>81</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/images/images/imago.png);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QTextBrowser" name="text_title">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>80</y>
            <width>111</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Noto Serif</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Serif'; font-size:18pt; font-weight:600; font-style:italic;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Imago&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="login_footer">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
